XenoHolographic-it/IAR-SDK-Sample-Android
# Feature/iar 4921

# Overview

- Surface view with screenshot/record functionality.

# Resources

- https://iversoft.atlassian.net/browse/IAR-4921
- https://iversoft.atlassian.net/browse/IAR-4922

# Technical Considerations

- Added a place/move button in similar vain as the screenshot and record buttons.

# Known Issues



# Testing Instructions

- Go to either on demand/location markers and click on any of the markers. It should bring you to the surface view
- Assert that the model is displaying
- Assert that clicking on the place button anchors the model
- Assert that clicking on screenshot/record takes a screenshot/records
- Assert that clicking on the move button unanchors the model.

# Screenshots
